app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new(like_params)
+    @like.author = current_user
+    @like.save
+    redirect_to "/users/#{flash[:post_author_id]}/posts/#{@like.post_id}"
+  end
```

Comment:
- [ ] in this method you should use the `respond_to` Rails helper method

Reply:
done!